src/js/view/view.js

```diff
@@ -811,10 +811,12 @@ define([
         }
 
         function _userActivity() {
-            _showing = true;
-            utils.removeClass(_playerElement, 'jw-flag-user-inactive');
+            if(!_showing){
```

Comment:
It's more efficient to use this with the _showing flag since we will only run that once when content becomes visible.   While we don't take a big performance hit on Chrome from `_controlbar.checkCompactMode(_videoLayer.clientWidth);` being run multiple times, it could be less efficient on other browsers.